categorization-svm-2/QuestionMark - with bubble.ipynb

```diff
@@ -0,0 +1,1483 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Implementation on new data with Bubble"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 262,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import re\n",
+    "import sys\n",
+    "import csv\n",
+    "import time\n",
+    "import json\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from tqdm import tqdm_notebook\n",
+    "from unidecode import unidecode\n",
+    "\n",
+    "from collections import defaultdict\n",
+    "from collections import Counter\n",
+    "from nltk import word_tokenize\n",
+    "from nltk.corpus import stopwords\n",
+    "\n",
+    "from sklearn.svm import LinearSVC\n",
+    "from sklearn.metrics import classification_report\n",
+    "from sklearn.metrics import precision_recall_fscore_support\n",
+    "from sklearn.metrics import recall_score\n",
+    "from sklearn.multiclass import OneVsRestClassifier\n",
+    "from sklearn.cross_validation import cross_val_score\n",
+    "from sklearn.cross_validation import cross_val_predict\n",
+    "from sklearn.cross_validation import LeaveOneLabelOut\n",
+    "from sklearn.cross_validation import train_test_split\n",
+    "from sklearn.feature_extraction.text import CountVectorizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "STOPWORDS = '''\n",
```

Comment:
Suggestion: keep the feature extraction in a separate `features.py` file, and use something like
```python
from features import *
```
Then you can just use the function names, without having to duplicate the code here. Not a big thing, but it helps to keep the code a bit cleaner.